pkg/datastore/datastore.go

```diff
@@ -0,0 +1,112 @@
+package datastore
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+)
+
+// Get gets a fqField from the datastore.
+func Get(ctx context.Context, addr string, key string, value interface{}) error {
+	reqBody, err := getRequestBody(key)
```

Comment:
If this function is only used here I propose changing the structure: Use one file for get with the main Get() and some helper functions and one file for set (in the same way).

Reply:
I don't think this is necessary yet. Maybe later.